src/components/Navbar/styles.module.scss

```diff
@@ -0,0 +1,3 @@
+.Navbar {
+  justify-content: space-between;
```

Comment:
Since `justify-content` only works for flex, how about explicitly setting `display: flex`? Otherwise, it seems a bit risky to rely on an implicit flex display.

Reply:
Because this is a `react-bootstrap` component, we know it will have `display: flex` on it. That's the way these bootstrap classes work, which is why I left it off. But I see no reason not to explicitly set it, so I will.